Content.Shared/_Impstation/Kodepiia/Components/KodepiiaConsumeActionComponent.cs

```diff
@@ -0,0 +1,28 @@
+using Content.Shared.Damage;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._Impstation.Kodepiia.Components;
+
+[RegisterComponent, NetworkedComponent]
+public sealed partial class KodepiiaConsumeActionComponent : Component
+{
+    [DataField]
+    public EntityUid? ConsumeAction;
+    [DataField]
+    public string? ConsumeActionId = "ActionKodepiiaConsume";
+
+    public List<SoundSpecifier?> SoundPool = new()
```

Comment:
this is just a copy of the existing gib soundCollection - `public SoundSpecifier SoundPool = new SoundCollectionSpecifier("gib");`
would require a rewrite of `PlayMeatySound` in `KodepiiaConsumeSystem` though